hive-hooks/path-conversion-hook/src/main/java/com/expediagroup/apiary/extensions/hooks/pathconversion/config/Configuration.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (C) 2018-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.hooks.pathconversion.config;
+
+import static java.lang.String.format;
+
+import java.util.*;
```

Comment:
Please expand these.

Reply:
Sorry about the large number of import / linting errors, didn't realize my auto-fix settings in IntelliJ had stopped working 🙃 . These should all be fixed now.